### ISSUE TITLE
TypeError: Cannot read property 'iana' of undefined

### ISSUE THREAD

**Issue by runely** [jens-maus/node-ical#22]
After update from 0.11.3 to 0.12.0 the exception **TypeError: Cannot read property 'iana' of undefined** is thrown when parsing an exchange ics file.
Same ics file worked fine on 0.11.3

This is happening when using *async.fromURL* with options *{}*:
```javascript
const nodeIcal = require('node-ical');
let options = {};
nodeIcal.async.fromURL(url, options, (err, data) => {
    if (err) {
        console.log("Failed:", err);
    }

    console.log("Success:", data);
});
```

If options is not given it works, but output is string and not json
```javascript
const nodeIcal = require('node-ical');
nodeIcal.async.fromURL(url, (err, data) => {
    if (err) {
        console.log("Failed:", err);
    }

    console.log("Success:", data);
});
```

Exception:
```javascript
/Users/runely/source/nodeical-test/node_modules/node-ical/ical.js:108
  return zoneTable[msTZName].iana[0];
                             ^

TypeError: Cannot read property 'iana' of undefined
    at getIanaTZFromMS (/Users/runely/source/nodeical-test/node_modules/node-ical/ical.js:108:30)
    at /Users/runely/source/nodeical-test/node_modules/node-ical/ical.js:175:23
    at Object.DTSTART (/Users/runely/source/nodeical-test/node_modules/node-ical/ical.js:457:32)
    at Object.handleObject (/Users/runely/source/nodeical-test/node_modules/node-ical/ical.js:483:39)
    at Object.parseLines (/Users/runely/source/nodeical-test/node_modules/node-ical/ical.js:534:18)
    at Object.parseICS (/Users/runely/source/nodeical-test/node_modules/node-ical/ical.js:587:12)
    at Request._callback (/Users/runely/source/nodeical-test/node_modules/node-ical/node-ical.js:111:12)
    at Request.self.callback (/Users/runely/source/nodeical-test/node_modules/request/request.js:185:22)
    at Request.emit (events.js:314:20)
    at Request.<anonymous> (/Users/runely/source/nodeical-test/node_modules/request/request.js:1154:10)
```

I've set ut a repo where you can test it out, you just need to get your own Exchange calendar url: https://github.com/runely/nodeical-test

**Comment by jens-maus** [comment]
Thanks for the report, however, the issue was already fixed in #12 and I also just published 0.12.1 which should fix it for everyone.

**Comment by runely** [comment]
0.12.1 fixed it 👍
